.ci-scripts/x86-64-apple-darwin-release.bash

```diff
@@ -39,8 +39,8 @@ ASSET_DESCRIPTION="https://github.com/ponylang/ponyc"
 
 # Build pony installation
 echo "Building ponyc installation..."
-make install prefix=${BUILD_PREFIX} default_pic=${PIC} arch=${ARCH} \
-  -j${MAKE_PARALLELISM} -f Makefile-lib-llvm symlink=no
```

Comment:
`symlink=no` isn't needed anymore?

Reply:
I have made the Makefile set `symlink=no` if `DESTDIR` is specified.   That seemed like what the logic in the original makefile was doing.  I.e. if `prefix` is specified it'll be something like `/usr/local` or `~/.local` and it's better install to `$prefix/lib/pony/$version` and symlink, but if `DESTDIR` is specified then you just want it there.

Reply:
gotcha.